modules/openxr/openxr_api.cpp

```diff
@@ -2077,7 +2076,7 @@ XRPose::TrackingConfidence _transform_from_location(const T &p_location, Transfo
 	Basis basis;
 	Vector3 origin;
 	XRPose::TrackingConfidence confidence = XRPose::XR_TRACKING_CONFIDENCE_NONE;
-	const auto &pose = p_location.pose;
+	const XrPosef &pose = p_location.pose;
```

Comment:
I had to look up what types `T` can take, where it's defined in `openxr.h`, to know which type its `pose` member uses. This really should be explicit for the reader.